src/Integration/Binding/ProjectRuleSetWriter.cs

```diff
@@ -74,10 +79,17 @@ private bool TryUpdateExistingProjectRuleSet(string solutionRuleSetPath, string
             }
 
             existingRuleSetPath = PathHelper.ResolveRelativePath(currentRuleSet, projectRuleSetRootFolder);
+
```

Comment:
Could the returned path ever be in a non-canonical form (i.e. containing . or ..)?